__tests__/ordCdsrc.test.js

```diff
@@ -39,12 +39,16 @@ describe("Tests for default ORD document when .cdsrc.json is present", () => {
 
         test("PartOfPackage values are valid ORD IDs ", () => {
             for (const apiResource of document.apiResources) {
+                if (!apiResource.partOfPackage) continue;
```

Comment:
@zongqichen : In case of the `injected` resource(the one given in `__tests__\bookshop\ord\custom.ord.json`), where no `partOfPackage` is stated, this check should be skipped.